packages/react-ui/storybook/stories/molecules/Alert.stories.js

```diff
@@ -0,0 +1,260 @@
+import React from 'react';
+import { Box, Button, Typography, capitalize, styled } from '@mui/material';
+import { Standalone } from '../../utils/storyStyles';
+import { Alert } from '@carto/react-ui';
```

Comment:
Please, import the Alert with a relative path, so this way we don't need to run `yarn build `every time a change is done there... Just to be faster for development